debian/rules

```diff
@@ -34,6 +34,9 @@ override_dh_auto_test:
 
 override_dh_auto_install:
 	DESTDIR=$(CURDIR)/debian/tmp ninja -C build-deb install
+	# Debian kmod has a conf file called mlx4.conf that unnecessarily
+	# tries to autoload mlx4_en, call our conf file rdma-mlx4.conf instead
+	mv $(CURDIR)/debian/tmp/etc/modprobe.d/mlx4.conf $(CURDIR)/debian/tmp/etc/modprobe.d/rdma-mlx4.conf
```

Comment:
Okay then.

@talatb To be clear, the mlx4.conf in kmod is *not* replaced by the one in rdma-core, they do totally different things.

The mlx4.conf in kmod is attempting to autoload ethernet support. I believe that the kernel now does that automatically via request_module.

Also, I haven't moved redhat/rdma.mlx4.sys.modprobe (which does the mlx4-setup thing) into common code because Mellanox said they prefer to do this setup another way. If so I recommend removing it from redhat as well.